src/client/failure/policies.ts

```diff
@@ -264,6 +264,9 @@ export class RetryPolicy extends FailurePolicy {
   }
 
   override afterFailure(info: ExecutionFailure): FailureResolution {
+    if (info.kind === 'bind') {
+      this.streak = 0;
```

Comment:
Shouldn't this set the streak to maximum to force an abort?

Reply:
Yep, i was handling this also in CircuitBreaker so this code never got used. Now it is handled in inner policy.